mqtt/mqtt-broker/src/connection.rs

```diff
@@ -1,8 +1,8 @@
-use std::fmt;
 use std::net::SocketAddr;
 use std::sync::Arc;
 use std::time::Duration;
 
+use derive_more::Display;
```

Comment:
Are we only using this to get a `Display` derive? I'm not sure taking on another dependency is worth it?

Reply:
I think yes, we could use it for something else. It has more features